test/POD/svd_burgers.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from read_hdf5 import *
+
+T = 1.0
+nDTout = 100
+DT = T/float(nDTout)
+
+archive = Archiver.XdmfArchive(".","burgers_3d",readOnly=True)
+
+import numpy as np
+
+u = read_from_hdf5(archive.hdfFile,'/u0')
```

Comment:
This bit I should be able to refactor using deim_utils to generate the SVD for  the u and F snapshots at the same time.